Classes/Hooks/Form.php

```diff
@@ -45,7 +45,7 @@ public function afterInitializeCurrentPage(FormRuntime $runtime, ?Page $currentP
             $expirationTime = (string)$this->getPageExpirationTime($runtime);
             // Set delay for initial form (no delay for re-submission of form)
             $extensionSettings = GeneralUtility::makeInstance(ExtensionConfiguration::class)->get('form_crshield');
-            $delay = $runtime->getFormSession() === null ? (int)($extensionSettings['crJavaScriptDelay'] ?? 3) : 0;
+            $delay = $runtime->getFormSession() === null ? intval($extensionSettings['crJavaScriptDelay'] ?? 3) : 0;
```

Comment:
I prefer using typecasting with (int), which is about 2-3 times faster than using intval().